pylint/extensions/_check_docs_utils.py

```diff
@@ -58,10 +58,19 @@ def possible_exc_types(node):
     """
     excs = []
     if isinstance(node.exc, astroid.Name):
-        excs = [node.exc.name]
+        inferred = safe_infer(node.exc)
+        if inferred:
```

Comment:
Yes, we should not report at all. Uninferable has now a false value, since yesterday, so the way you handled it should be enough.